src/Modules/CalcSections.lua

```diff
@@ -1601,6 +1601,28 @@ return {
 			{ modName = { "GuardAbsorbRate", "ChaosGuardAbsorbRate", "GuardAbsorbLimit", "ChaosGuardAbsorbLimit" } },
 		},
 	},
+	{ label = "Prevented life loss", haveOutput = "AnyPreventedLifeLoss",
```

Comment:
can you just capitalize this to `Prevented Life Loss` to keep format with other rows in that table please?

Reply:
Sure, updated